### DISCUSSION TITLE
Can't select the domain -Exception while executing function: GetDnsZones_Orchestrator Orchestrator function 'GetDnsZones_Orchestrator' failed: The activity function 'GetZones' failed: "Specified method is not supported.". See the function execution logs for additional details.

### DISCUSSION THREAD

**Discussion by romina2001** [polymind-inc/acmebot#263] [Q&A] [answered]
Hello,

When I am trying to reach to the GUI interface I ger the following 
![image](https://user-images.githubusercontent.com/53087771/108078142-fa6f1300-7075-11eb-862b-2fad7c72e0b2.png)

Than it disappears and I can't select the certificate. I am missing something? 
![image](https://user-images.githubusercontent.com/53087771/108078089-eaefca00-7075-11eb-895f-9063a4ca7d98.png)

**Comment by shibayan** [comment, marked as answer]
Similar #260

**Reply by romina2001** [reply]
All good, I fixed it. Thanks
